app/assets/javascripts/darkswarm/controllers/offcanvas_controller.js.coffee

```diff
@@ -0,0 +1,11 @@
+Darkswarm.controller "OffcanvasCtrl", ($scope, $window) ->
+  $scope.menu = $(".left-off-canvas-menu")
+
+  $scope.setOffcanvasMenuHeight = ->
+    $scope.menu.height($(window).height())
+
+  $scope.bind = ->
+    $(window).on("resize", $scope.setOffcanvasMenuHeight)
```

Comment:
looking at this now I remembered this: https://www.tutorialsteacher.com/angularjs/angularjs-window-service
They say it's better for testing. Currently we have 4 cases of $(window) and a few more of $window

and the same for $(".left-off-canvas-menu") where we have a mix of $('') and angular.element

It's not very important but maybe we can define a best practice and do it one way or another.

Reply:
@luisramos0 Ah, yes about `angular.element`, I should have done that.

About `$window`, I might be wrong but, from what I understand, it is not a full-featured equivalent of `window` but only has properties that we would normally want to stub in tests.

I initially used `$window.height`, but it turns out is not supported (`undefined`). I forgot to remove `$window` afterwards. This removes it: https://github.com/openfoodfoundation/openfoodnetwork/pull/3786/commits/dae23d3c3ba970b47c4ded0d8a3c2a739b051b7f

Thanks, @luisramos0!

Reply:
ah  cool, I didnt notice $window was in the deps list :-) 👍